modules/saml/www/sp/saml2-acs.php

```diff
@@ -51,14 +53,17 @@
     }
 }
 
-$idp = $issuer;
```

Comment:
This means we have to test both $idp and $issuer for all possible types.. Therefore I've renamed all instances of `$idp` to `$issuer`